specs/IsSmartScreenRequired.md

```diff
@@ -0,0 +1,98 @@
+# Background
+
+[Edge SmartScreen](https://support.microsoft.com/en-us/microsoft-edge/how-can-smartscreen-help-protect-me-in-microsoft-edge-1c9a874a-6826-be5e-45b1-67fa445a74c8) helps end users identify reported phishing and malware websites, and also helps end users make informed decisions about downloads.
+
+Currently, developers can use `options->put_AdditionalBrowserArguments(L"--disable-features=msSmartScreenProtection")` to disable SmartScreen in the WebView2 application. It is a startup parameter of the browser process and applies to all WebView2 instances using the same user data folder. It must be determined when the WebView2Environment is created, and it cannot be modified at runtime.
+
+To support more flexibility we introduce a new API.
```

Comment:
(Don't forget to update [this page](https://docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2#smartscreen), which currently documents and recommends the `--disable-features=msSmartScreenProtection` technique.)

Reply:
Ok, I have contacted the corresponding team to update the documentation